src/bundler.ts

```diff
@@ -125,18 +125,54 @@ class Bundler {
     return false;
   }
 
-  hide(node) {
+  /**
+   * Creates a hidden container <div> to which inlined content will be
+   * appended.
+   * TODO(usergenic): Give this a more intention-revealing name.
+   */
+  getHiddenNode(): ASTNode {
     const hidden = dom5.constructors.element('div');
     dom5.setAttribute(hidden, 'hidden', '');
     dom5.setAttribute(hidden, 'by-vulcanize', '');
-    this.removeElementAndNewline(node, hidden);
-    dom5.append(hidden, node);
+    return hidden;
+  }
+
+  /**
+   * Inline external scripts <script src="*">
```

Comment:
Where is this script inlined? From the parameter names, I'm having trouble which ones might be mutated.